django_mailbox/models.py

```diff
@@ -347,12 +347,12 @@ def _get_dehydrated_message(self, msg, record):
                 )
             except ValueError:
                 logger.warning(
-                    "Decoding error encountered; interpreting as ASCII!",
+                    "Decoding error encountered; interpreting %s as ASCII!",
                     content_charset
                 )
                 msg.set_payload(
                     msg.get_payload(decode=True).decode(
-                        content_charset,
+                        'ascii',
```

Comment:
Hrm; I wonder why I lied before; we definitely weren't interpreting this as ascii.